src/level.js

```diff
@@ -19,22 +19,11 @@ async function readLevel (path) {
   return nbt.simplify(dnbt).Data
 }
 
-async function writeLevel (path, value) {
-  const nbt = {
+async function writeLevel (path, levelData) {
+  const data = await writeAsync({
```

Comment:
can you make this an nbt.comp() with a name property?

Reply:
Something like that ?
```javascript
const nbt = nbt.comp({ Data: levelData })
nbt.name = ''
const data = await writeAsync(nbt)
```

Reply:
yeah